infra-templates/infoblox/0/rancher-compose.yml

```diff
@@ -0,0 +1,63 @@
+# notemplating
+.catalog:
+  name: "Infoblox DNS"
+  version: "v0.1.0"
+  description: "Rancher External DNS service powered by Infoblox"
+  minimum_rancher_version: v1.2.0-pre4-rc1
```

Comment:
Is this really the right min version to use?

Reply:
@deniseschannon Hi Denise, this min version just the same as other DNS provider, we tested the infoblox DNS provider in rancher 1.4 and 1.6, how about we change to 1.4?

Reply:
Can you please update it to `v1.6.0`?

Reply:
Just updated to v1.6.0, thanks.